sdk/servicebus/Azure.Messaging.ServiceBus/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Release History
 
-## 7.0.0-preview.10 (Unreleased)
-
+## 7.0.0-preview (2020-11-23)
+### Breaking Changes
```

Comment:
`-preview` and I didn't notice why it is `7.0.0` until now. Shouldn't it be `1.0.0`?
Also, there shouldn't be breaking changes on a Release changelog (not applicable to a release).

Reply:
No we are using 7.0.0.

What do you mean - those changes are related to this release.